frontend/javascripts/oxalis/view/components/input_component.tsx

```diff
@@ -96,19 +119,31 @@ class InputComponent extends React.PureComponent<InputComponentProps, InputCompo
 
   render() {
     const { isTextArea, onPressEnter, title, style, ...inputProps } = this.props;
-    const InputComponentType: typeof TextArea | typeof Input = isTextArea ? TextArea : Input;
-    const input = (
-      <InputComponentType
+    // const InputComponentType: typeof TextArea | typeof Input = isTextArea ? TextArea : Input;
+    const input = isTextArea ? (
+      <TextArea
+        {...inputProps}
+        style={title == null ? style : undefined}
+        onChange={this.handleChange}
+        onFocus={this.handleFocus}
+        onBlur={this.handleBlur}
+        value={this.state.currentValue}
+        onPressEnter={onPressEnter}
```

Comment:
why no `blurYourself` anymore as a fallback?

Reply:
Hehehe, good catch. I removed it so the review can find something ;-) Will restore it.